### DISCUSSION TITLE
svg images creation

### DISCUSSION THREAD

**Discussion by keybasegit** [nunocoracao/blowfish#1512] [General]
### Thanks very much for your generous support 🙏🏻 Faust

**Comment by nunocoracao** [comment]
Hey, tbh I didn't keep track of it. I just searched for svg animation generator and went through a couple of them...

**Comment by knowinginner** [comment]
### Thanks for your answer dear Nunocoracao. Take care. Vlad

**Comment by v01tt** [comment]
Hey Nuno, how do you generate the animated background.svg? Is there a particular program you use or website you can find them at?
Would really appreciate it if you could share a bit on the process of making these types of svg.
